src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/yml/templating_no_assets.yml

```diff
@@ -1,3 +1,5 @@
 framework:
+    assets:
+        enabled: true
```

Comment:
passing `null` instead of the nested `enabled` key does not have the desired effect, this seems to be unintentionally broken by the changes made in #25789